CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 ### Changes
 
 * [#8487](https://github.com/rubocop-hq/rubocop/pull/8487): Detect `<` and `>` as comparison operators in `Style/ConditionalAssignment` cop. ([@biinari][])
+* [#8362](https://github.com/rubocop-hq/rubocop/issues/8362): Add numbers of correctable offenses to summary. ([@nguyenquangminh0711][])
```

Comment:
You'll have to move this under `master`, as I just cut 0.89.1.

Reply:
Done. Sorry, I'm not aware that you cut a new relase some couple of hours ago